samples/csharp/MicrosoftEduGraphSamples/Workflows/SubmissionWorkflow.cs

```diff
@@ -60,32 +61,65 @@ public async Task ReassignWorkflow()
                     throw new Exception($"No submission found for {_config["studentAccount"]} in {assignmentId} for class {_config["classId"]}");
                 }
 
+                // Get submission outcomes
+                var submissionOutcomes = await Submission.GetSubmissionOutcomes(
```

Comment:
GetSubmissionOutcomesAsync

Reply:
renamed "GetSubmissionOutcomes" to "GetSubmissionOutcomesAsync"